cc/model/features.h

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CC_MODEL_FEATURES_H_
+#define CC_MODEL_FEATURES_H_
+
+#include <array>
+#include <utility>
+#include <vector>
+
+#include "cc/color.h"
+#include "cc/constants.h"
+#include "cc/logging.h"
+#include "cc/model/features_internal.h"
+#include "cc/model/types.h"
+#include "cc/platform/utils.h"
+#include "cc/position.h"
+
+namespace minigo {
+
```

Comment:
an overview of these features & the variadic scheme belongs somewhere, either here or in internal.h